.vscode/configurations/windows-docker/tasks.json

```diff
@@ -18,7 +18,7 @@
         {
             "type": "shell",
             "label": "1. Copy source code to docker volume",
-            "command": "docker exec -t tyra-game-compiler sh -c 'rsync -ac --delete --exclude \".git\" --exclude \".vscode\" --exclude \"*obj*/**\" --exclude \"*bin*/**\" /host/ /src/'"
+            "command": "docker exec -t tyra-game-compiler sh -c 'rsync -ac --delete --exclude \".git\" --exclude \".vscode\" --exclude \"obj/**\" --exclude \"bin/**\" /host/ /src/'"
```

Comment:
There was 2 problems with docker:
1 - `res/` folder was not copied to output directory. This was a Tyra bug, so please run "Update Tyra" task

2 - There was problem with copying binaries (problem `file has vanished..`), so please apply these configuration patches

Reply:
I saw that bugs, but I thougth I did changed something I soudn't... Thx for fixing!